src/spec-configuration/containerFeaturesOrder.ts

```diff
@@ -87,19 +88,19 @@ export function computeInstallationOrder(features: FeatureSet[]) {
         }
         orderedFeatures.push(
             ...current.map(node => node.feature)
-                .sort((a, b) => a.features[0].name.localeCompare(b.features[0].name)) // stable order
+                .sort((a, b) => a.sourceInformation.userFeatureId.localeCompare(b.sourceInformation.userFeatureId)) // stable order
```

Comment:
This was there before, but I think using `localeCompare` is not a very good idea, as it might be influenced by the locale of the current machine running the code. So `u` and `ü` might be sorted differently on different machines with different locales.

Reply:
Something to note is that this will also likely change the execution order for users, which may have unintended effects.  This auto ordering is by-design though, and unavoidable right now

Reply:
@alexdima Switched to 👇 , should that help address the issue? 

> .sort((a, b) => a.sourceInformation.userFeatureId < b.sourceInformation.userFeatureId ? -1 : 1)

Reply:
:+1: Yes, switching to use `<` guarantees a stable order across different machines, as it is not locale dependent. See https://github.com/microsoft/vscode/issues/70920#issuecomment-546405072 for more context.